SwiftLeeds/Views/My Conference/ActivityView.swift

```diff
@@ -25,7 +25,7 @@ struct ActivityView: View {
         VStack(spacing: Padding.stackGap) {
             HeaderView(
                 title: activity.title,
-                imageURL: URL(string: "\(BaseURL.image)\(activity.image ?? "")"),
```

Comment:
As it's no longer concatenating values, no longer need the ?? ""

Reply:
We do unfortunately, the `activity.image` is the cause of the optionality here, not the concat 👍🏻

Reply:
Ah of course, that's a shame